Index.html

```diff
@@ -75,14 +75,61 @@
                         <input type="text" placeholder="Search GitHub">
                     </li>
                     <li><a href="#">Sign in</a></li>
-                    <li><button class="button"><a href="#">Sign up</a></button></li>
+                    <li><button class="button1"><a href="#">Sign up</a></button></li>
```

Comment:
Try to avoid using numbers in css classes, i'd probably call this `button_one` or `button-one`